### DISCUSSION TITLE
DeckGL getCursor callback

### DISCUSSION THREAD

**Discussion by SchemeSonic** [visgl/deck.gl#6547] [Q&A] [answered]
Hello everyone,

 I've noticed that getCursor callback is triggered continuously whether move mouse or not. So the question is, is there any way to change the frequency of the callback triggers? (Like triggering callback only for mouse actions etc.)

thanks 

```typescript
<DeckGL
  controller={true}
  initialViewState={viewport}
  getCursor={(args) => console.log(args)}
>
```

**Comment by Pessimistress** [comment, marked as answer]
No. You should not perform any expensive operation in this callback.
